tests/unit/libbpf_test.cpp

```diff
@@ -3361,21 +3361,26 @@ TEST_CASE("bind_tail_call_max_exceed", "[libbpf]")
 }
 
 void
-_test_batch_iteration_maps(fd_t& map_fd, uint32_t batch_size, bpf_map_batch_opts* opts)
+_test_batch_iteration_maps(
+    fd_t& map_fd, uint32_t batch_size, bpf_map_batch_opts* opts, size_t value_size, int num_of_cpus)
 {
     // Retrieve in batches.
     const uint8_t batch_size_divisor = 10;
     uint32_t* in_batch = nullptr;
     uint32_t out_batch = 0;
     std::vector<uint32_t> returned_keys(0);
     std::vector<uint64_t> returned_values(0);
-    uint32_t requested_batch_size = (batch_size / batch_size_divisor) - 2;
+    int32_t requested_batch_size = (batch_size / batch_size_divisor) - 2;
     uint32_t batch_size_count = 0;
     int result = 0;
 
+    if (requested_batch_size <= 0) {
```

Comment:
Can you explain why this change was needed? Are we hitting this now because of changes in this PR?